doc/source/api/index.rst

```diff
@@ -4,33 +4,34 @@
 API reference
 =============
 
-This section describes the core Pythonic interfaces for Twin Runtimes.
-Here, you can find the application programming interfaces for 
-using the core Twin Runtime SDK functionalities, as well as an example
-of higher level abstraction class to evaluate the Runtimes.
+This section describes the core Pythonic interfaces for twin runtimes.
+Here, you can find the APIs for using twin runtime functionalities and
+the higher-level abstraction class for evaluating twin runtimes.
```

Comment:
@lboucin I am not sure I understand the distinction made in lines 8 and 9 between using twin runtime functionalities and evaluating twin runtimes. Is there a better way to explain what is intended here?

Reply:
@PipKat We want to emphasize the distinction between the TwinRuntime and TwinModel APIs. Would following variation make more sense? "Here, you can find all APIs for consumming twin runtimes, from the lowest-level APIs with the :class:`TwinRuntime <ansys.pytwin.TwinRuntime` class to the higher-level with the :class:`TwinModel <ansys.pytwin.TwinModel` class."

Reply:
@lboucin I like this much better and changed the file! Thanks!